client/src/assets/colors.scss

```diff
@@ -0,0 +1,8 @@
+$white: #ffffff;
+$black: #000000;
+$crimson: #d72130;
+$rose-white: #fffbfb;
+$shuttle-gray: #596377;
+$river-bed: #434c5d;
+$mischka: #d4d9e2;
+$monza: #c80515;
```

Comment:
it is better to call variables than colors. Because there can be some width or other values